package.json

```diff
@@ -35,7 +35,7 @@
     "atom-package-deps": "^4.0.1",
     "cosmiconfig": "^1.1.0",
     "deep-assign": "^2.0.0",
-    "stylelint": "5.2.1",
+    "stylelint": "NEEDS UPDATING WHEN AVAILABLE",
```

Comment:
Don't put this in, we'll simply not merge this until `stylelint` has been updated.